src/common/mongo/manager.go

```diff
@@ -72,8 +72,13 @@ func connectDB(options map[string]string) (session *mgo.Session, err error) {
 		url += urlServers
 		// url += ("/" + dbname)
 	}
-
-	log.Debugf("Mongo URL: %s", url)
+	// filter out the MongoDB secrets
+	result := strings.Split(url, "@")
+	if len(result) > 1 {
+		log.Debugf("Mongo URL: %s", result[1])
```

Comment:
@zkf9971 Does result[1] include the whole URL excluding credentials?

Reply:
@ypjin Right. in this case, the excluded result[0] includes credentials